conceptnet5/nodes.py

```diff
@@ -38,6 +41,86 @@
 }
 
 
+def standardize_as_list(text, token_filter=None):
```

Comment:
token_filter is undocumented. The documentation seems to suggest that token_filter should instead be a flag to apply english_filter.